### DISCUSSION TITLE
ImFontGlyphRangesBuilder

### DISCUSSION THREAD

**Discussion by project224** [ocornut/imgui#5790] [New Users Build/Link/Run issues ONLY!]
i am trying to add support for these chars "šđčćžŠĐČĆŽ"
code is called before imgui_app;

GetGlyphRangesDefault returns these ranges
0x0020, 0x00FF,
0x0100, 0x017F, // [Latin Extended-A](https://jrgraphix.net/r/Unicode/0100-017F)

    ImVector<ImWchar> ranges;
    ImFontGlyphRangesBuilder builder;
    builder.AddRanges(io.Fonts->GetGlyphRangesDefault()); 
    builder.BuildRanges(&ranges);                        
   
    io.Fonts->AddFontFromFileTTF("C:\\Windows\\Fonts\\Tahoma.ttf", 18, NULL, ranges.Data);
    io.Fonts->Build();'

rendered text
![image](https://user-images.githubusercontent.com/115223468/196401030-0138b766-0b7d-4021-b63a-085134ce3869.png)

**Comment by ocornut** [comment]
See 
https://github.com/ocornut/imgui/blob/master/docs/FAQ.md#q-i-integrated-dear-imgui-in-my-engine-and-little-squares-are-showing-instead-of-text

And use tools such as the ones mentioned here:
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md

Perhaps you are loading a font but texture hasn't been updated in the Backends.
Please be more specific with what works and what doesn't to get help. Thank you.

**Reply by project224** [reply]
is calling font code before imgui_app fn correct?;

**Comment by ocornut** [comment]
I don’t know what imgui_app is.

**Reply by project224** [reply]
sorry; https://github.com/pplux/imgui-app it might be the issue with this project

**Comment by ocornut** [comment]
Yes please look at imgui_app source code or ask them about it.
